third_party/CMakeLists.txt

```diff
@@ -19,6 +16,15 @@ if(${EFFCEE_ENABLE_TESTS})
 endif()
 
 if (IS_DIRECTORY ${EFFCEE_RE2_DIR})
+
+  if ("${MSVC}")
```

Comment:
I just updated the docs to describe how you can use CMake vars to point to alternate locations for third party dirs.
It can be even more automatic..  I'll take a look.